File: src/targets/ocaml/cohttp/fixtures/query.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

<<<<<<< HEAD
let uri = Uri.of_string "https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value" in
=======
let uri = Uri.of_string "http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value" in
>>>>>>> upstream/master

Client.call `GET uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: src/targets/clojure/clj_http/fixtures/query.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/get "https://httpbin.org/anything" {:query-params {:foo ["bar" "baz"]
                                                           :baz "abc"
                                                           :key "value"}})
=======
(client/get "http://mockbin.com/har" {:query-params {:foo ["bar" "baz"]
                                                     :baz "abc"
                                                     :key "value"}})
>>>>>>> upstream/master
